doc/installation_instructions/basic.rst

```diff
@@ -270,9 +276,25 @@ Install SmartSim from Source
 
 First, clone SmartSim.
 
-.. code-block:: bash
+.. tabs::
+
+  .. tab:: Linux
+
+    .. code-block:: bash
+
+      git clone https://github.com/CrayLabs/SmartSim smartsim
+
+  .. tab:: MacOS (Intel x64)
+
+    .. code-block:: bash
+
+      git clone https://github.com/CrayLabs/SmartSim smartsim
+
+  .. tab:: MacOS (Apple Silicon)
 
-  git clone https://github.com/CrayLabs/SmartSim smartsim
+    .. code-block:: bash
+
+      git clone --config core.autocrlf=true https://github.com/CrayLabs/SmartSim smartsim
```

Comment:
I think I may have also beeen forced to ensure that non-mac gcc came before mac version in path. must verify...